eng/scripts/TypeSpec-Requirement.ps1

```diff
@@ -8,9 +8,60 @@ param (
 )
 Set-StrictMode -Version 3
 
```

Comment:
Can we also add some documentation somewhere like the ci-fix.md so we can point folks to that when they need to add a suppression.

Reply:
It should be documented here:
https://aka.ms/azsdk/pr-suppressions